.circleci/config.yml

```diff
@@ -36,40 +15,28 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-          - v3-rbenv-dependencies-20230418-{{ checksum ".ruby-version" }}
-          - v3-dependencies-v3-{{ checksum "Gemfile.lock" }}
-
-      - run:
-          name: rbenv update
-          command: |
-            rbenv_update_env
-
-      - save_cache:
-          paths:
-            - ~/.rbenv
-          key: v3-rbenv-dependencies-20230418-{{ checksum ".ruby-version" }}
+          - bundle-{{ checksum "Gemfile.lock" }}
 
       - run:
           name: bundle install
           command: |
             # required for the license_finder
-            # The Gemfile.lock specifies nokogiri (1.14.3)
+            # The Gemfile.lock specifies nokogiri (1.16.2)
             # but bundler would install native gem instead if not forced
             bundle config set --local force_ruby_platform true
-            bundle_install_rubies
+            bundle config set --local path ./vendor/bundle
+            bundle install
 
       - save_cache:
           paths:
-            - ~/.rbenv/versions
-          key: v3-dependencies-v3-{{ checksum "Gemfile.lock" }}
+            - ./vendor/bundle
+          key: bundle-{{ checksum "Gemfile.lock" }}
 
-      #run tests!
       - run:
-          name: Run tests on Ruby 3.0.2
-          command: |
-            TEST_RUBY_VERSION=3.0.2 script/ci
-
-      #- run:
-      #    name: Run tests on all Rubies
-      #    command: |
-      #      TEST_ALL_RUBIES=1 script/ci
+          name: Run tests
+          command: bundle exec script/test
+workflows:
+  version: 2
+  workflow:
+    jobs:
+    - build
```

Comment:
:eyes:

Reply:
🤔

Reply:
@jlledom new line missing at the end of the file :sweat_smile:

Reply:
OMG!!! How can this happen to me??